python/lib.w

```diff
@@ -112,3 +114,30 @@ pub class InflightBucketEvent impl cloud.IBucketEventHandler {
     return this;
   }
 }
+
+pub class InflightApiOnRequest impl cloud.IApiEndpointHandler {
+  _inflightType: str;
+  inner: types.IApiOnRequest;
+
+  new(props: types.InflightProps) {
+    this._inflightType = "_inflightPython";
+
+    let target = util.env("WING_TARGET");
```

Comment:
@Chriscbr @hasanaburayyan what are we doing to get rid of this pattern?

Do we have a planned design? Do we have this in our tasks?